base/config/quality/detekt-config.yml

```diff
@@ -0,0 +1,9 @@
+naming:
```

Comment:
This looks like settings that are applicable to all subprojects. So, they should probably go under `./config/quality`. I mean be not under submodule, but under `config` of the project root.